software/chipwhisperer/capture/acq_patterns/basic.py

```diff
@@ -27,7 +27,7 @@
 from ...common.utils import util
 from ._base import AcqKeyTextPattern_Base
 from ...common.utils.util import camel_case_deprecated
-bytearray = util.bytearray # type: ignore
+bytearray = util.CWByteArray # type: ignore
```

Comment:
Lines 190 and 209 are using `util.bytearray()`, which needs to be changed to `bytearray()`

Reply:
Fixed! Also grepped the project for other similar instances and fixed those, as well.